src/vm/i386/asmhelpers.S

```diff
@@ -0,0 +1,7 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I would prefer not adding dummy asm files. We should add them once / if they are needed.